src/main/java/org/sagebionetworks/bridge/exporter/worker/TsvInfo.java

```diff
@@ -37,6 +38,7 @@ private TsvInfo() {
         this.file = null;
         this.writer = null;
         this.initError = true;
+        this.recordIds = null;
```

Comment:
I guess I would just this.recordIds = new ArrayList<>() right here, instead of requiring callers to pass it in.